aptos-move/framework/aptos-framework/sources/block.move

```diff
@@ -4,10 +4,14 @@ module aptos_framework::block {
     use std::features;
     use std::vector;
     use std::option;
+    use std::signer::address_of;
```

Comment:
style: Let's not import functions. You can just import std::signer here and use signer::address_of where needed

Reply:
fixed